README.md

```diff
@@ -67,13 +67,19 @@ MAX_API_SECRET=
 FTX_API_KEY=
 FTX_API_SECRET=
 # specify it if credentials are for subaccount
-FTX_SUBACCOUNT_NAME=
+FTX_SUBACCOUNT=
```

Comment:
thanks~

Reply:
oh it should be `FTX_API_SUBACCOUNT`

Reply:
It should be `FTX_SUBACCOUNT` actually https://github.com/c9s/bbgo/blob/main/pkg/cmd/cmdutil/exchange.go#L51

Reply:
oh you are right

Reply:
do you think we should the API to the prefix?